dkpro-core-io-brat-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/io/brat/BratWriter.java

```diff
@@ -562,7 +598,9 @@ private boolean isSlotFeature(FeatureStructure aFS, Feature aFeature)
     private void writeRelationAnnotation(BratAnnotationDocument aDoc, FeatureStructure aFS)
     {
         RelationParam rel = parsedRelationTypes.get(aFS.getType().getName());
-        
+        if (rel== null ) {// then is the parent type
+        	rel=parsedRelationTypes.get(aFS.getCAS().getTypeSystem().getParent(aFS.getType()).getName());
```

Comment:
Why is it the parent type?

Reply:
If we reach this function the type or his parent is in parsdRelationTypes
if the first one fails then is the parent. 
There are soo many relation types that using the parent simplifies the definition.